### ISSUE TITLE
ngbTypeahead Chrome autocomplete

### ISSUE THREAD

**Issue by jamesbcn** [ng-bootstrap/ng-bootstrap#3372]
The documentation for the input selector ngbTypeahead states:

> The value for the autocomplete attribute for the <input> element. Defaults to "off" to disable the native browser autocomplete, but you can override it if necessary.

However suggestions are still appearing in Chrome. I have tried to remove this by including:

`autocomplete="off"`
`autocomplete="new-password"`
`autocomplete="randomText"`

None of these work.

**Comment by peterblazejewicz** [comment]
So that is true for any other library or interaction. In Chrome, the `input` needs to be within a `form` tag in order to respect `autocomplete` attribute. Otherwise Chrome UI always prefers users:
https://bugs.chromium.org/p/chromium/issues/detail?id=587466
The `ng-bootstrap` docs could be amended to reflect this. The autocomplete `off` works in:

```html
<form>
  <label for="typeahead-basic">Search for a state:</label>
  <input id="typeahead-basic" type="text" class="form-control" [(ngModel)]="model" [ngbTypeahead]="search"/>
</form>
```
Thanks!

**Comment by jamesbcn** [comment]
Resolved. Thanks

**Comment by jamesbcn** [comment]
I closed this too early. Auto-complete will be disabled in your code but ngModel will no longer work. ngModel requires a name tag in order to work. However the auto-complete will be reactivated. I got around this by giving the input a randomly generated name.

**Comment by peterblazejewicz** [comment]
Yes, apologies. I'd should post a from tag only, the details are not important here (as far as we comply the Angular library rules).
What do you mean by `reactivated`?

**Comment by jamesbcn** [comment]
By introducing the name tag, the auto-complete (which is disabled by ngbTypeahead) is reactivated/re-enabled/activated once again. It overrides ngbTypeahead.

**Comment by maxokorokov** [comment]
@jamesbcn please provide a minimal reproduction scenario with Stackblitz, if you don't mind. It should have been fixed in #2194. I can't reproduce it in Chrome at the moment.

We should probably update docs/examples indeed if `<form>` is required.

**Comment by benouat** [comment]
Closing as no update regarding a reproducible scenario via Stackblitz
